src/reducers/resourceHelpers.js

```diff
@@ -202,6 +205,53 @@ export const updateValueErrors = (state, valueKey) => {
     if (value.uri && !isValidURI(value.uri)) errors.push("Invalid URI")
   }
 
+  if (
+    propertyTemplate.type === "literal" &&
+    propertyTemplate.validationDataType ===
+      "http://www.w3.org/2001/XMLSchema/integer" &&
```

Comment:
Is this going to give a validation error for null/no value? (We don't want it to.)

Reply:
done.